### PR TITLE
Fix docs browser freeze due to example notebook.

### PR DESCRIPTION
**Pull request by yhtang**
Also tuned the RBF example notebook: reducing the learning rate to 0.05 for RBF leads to better results.

### CODE REVIEW

File: funfact/loss.py [hunk 1]
```diff
@@ -99,3 +99,8 @@ class KLDivergence(Loss):
     '''KL Divergence loss.'''
     def _loss(self, model, target):
         return ab.multiply(target, ab.log(ab.divide(target, model)))
+
+
+mse = MSE()
```
Comment (campsd) [inline funfact/loss.py:104]: What is the reason for reintroducing these?

Comment (yhtang) [reply funfact/loss.py:104]: In case if the user (or me in this case) wants to directly use a loss function as a *function*, without the need to instantiate it first. :grin:

**Review by campsd — APPROVED** [review]
